config/holes.toml

```diff
@@ -729,6 +729,28 @@ preamble = '''
     for example <b>15</b> should print <b>FizzBuzz</b>.
 '''
 
+['Flood Fill']
+category = 'Computing'
+links = [
+    { name = 'Wikipedia', url = '//en.wikipedia.org/wiki/Flooding_algorithm' },
+]
+experimental = -1
```

Comment:
It's `experiment` not `experimental`, it's meant to point to an issue ID to discuss the experiment, `-1` is just a placeholder for I don't have an issue.

Reply:
```suggestion
experiment = 697
```
Ah got it 👍

Reply:
ty